Dockerfile

```
@@ -6,7 +6,7 @@ RUN apk add --update --no-cache build-base
 RUN apk add --update --no-cache libffi-dev
 RUN apk add --update --no-cache zlib-dev
 RUN apk add --update --no-cache jpeg-dev
-RUN apk add --update --no-cache freetype
+RUN apk add --update --no-cache libfreetype6-dev
```

```suggestion
RUN apk add --update --no-cache freetype-dev
```
[`libfreetype6-dev`](https://pkgs.alpinelinux.org/packages?name=libfreetype6-dev&branch=edge) is not a valid package in the Alpine registry.
Like I've mentioned several times before, [`freetype-dev`](https://pkgs.alpinelinux.org/packages?name=freetype-dev&branch=edge) is the correct package name.